realtek-poe: Builds for all targets

howels opened the pull request:
Maintainer: @mrnuke @Hurricos 
Compile tested: realtek/rtl839x (hpe_1920-48g-poe 5d26507a73b166e6a47522478455bb95bc73672b
Run tested: realtek/rtl839x (hpe_1920-48g-poe 5d26507a73b166e6a47522478455bb95bc73672b

Description:

Currently realtek-poe is built for mips-4kec with the realtek/rtl838x target but not built for mips-24kc with the realtek/rtl839x target. The package builds ok but buildbot is not triggering the extra build and the package is missing from mips-24kc repos.

This flag should instruct build systems to build the package for all targets.

howels commented:
Missed the git config before committing. Will update signoff

howels commented:
Apologies, thought I'd updated the commit earlier but confirmed that the signoff is correct now. Please trigger workflows again.

hauke commented:
This works when we build also nonshared packages n the image build process. I am not sure if this is the case.

howels commented:
> Please just remove the realtek dependency. Then it gets build for all.
> 
> Your change only works when the image build process also takes the feed, but I do not know if this is the case.

Removed dependency on `TARGET_realtek` and reverted the `nonshared` flag.

howels commented:
@hauke please trigger workflow again.

Neustradamus commented:
@howels: Thanks :)
